angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.interfaces.ts

```diff
@@ -51,6 +51,11 @@ export interface ISprkDropdownChoice {
    * this is omitted in those cases.
    */
   value?: string;
+  /**
+   * If `default` is set to `true` and `dropdownType` is set to `informational`
+   * then `triggerText` will be automatically changed to this choice `value`
+   */
+  default?: boolean;
```

Comment:
```suggestion
  isDefault?: boolean;
``` 
Could you update this?

Reply:
Will do, thanks! Could you also check my comment in [related issue](https://github.com/sparkdesignsystem/spark-design-system/issues/3529#issuecomment-704789437)?